README.md

```diff
@@ -273,6 +273,18 @@ And you can change the JSON key that the serializer should use for a particular
 The `url` declaration describes which named routes to use while generating URLs
 for your JSON. Not every adapter will require URLs.
 
+## Pagination
+
+If you want pagination links in your response, specify it in the `render`
+
+```ruby
+  render json: @posts, pagination: true
```

Comment:
Great! @bf4 about the meta thing, it would work, but it would not follow [json-api conventions](http://jsonapi.org/format/#fetching-pagination).
I think the README might not be the best place for describing this feature, mostly because it's to a particular adapter (json-api) and also because would be awesome to have some examples. 
We are trying to consolidate our [new docs](https://github.com/rails-api/active_model_serializers/tree/master/docs), would be great if instead of adding this to the README you could open a new article there with this PR describing the feature, some examples and some of the gems that ppl can use in order to make it work.